apps/web/src/contexts/payment-context.tsx

```diff
@@ -151,7 +146,7 @@ export function usePaymentProvider({
     [t]
   )
   const [formErrors, setFormErrors] = useState<FormValidation>()
-  const [price, setPrice] = useState<string | null>()
+  const [price, setPrice] = useState<number>(0)
```

Comment:
If the initial value matches the desired type of the `useState` hook, no need to set it explicitly.

```suggestion
  const [price, setPrice] = useState(0)
```


We're not very consistent with this in our code, but trying to address that.